be.telenet.user.qsstv.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 hamradio lu9dce -->
+<component type="desktop-application">
+ <id>be.telenet.user.qsstv</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0</project_license>
+ <name>qsstv</name>
+ <summary>receiving slow-scan television and fax</summary>
+ <developer_name>Johan Maes ON4QZ</developer_name>
+ <url type="homepage">http://users.telenet.be/on4qz/qsstv/</url>
+ <description>
+  <p>
+   QSSTV is a program for receiving slow-scan television and fax.
```

Comment:
From Fedora:
QSSTV is a program for receiving and transmitting SSTV and HAMDRM (sometimes called DSSTV). It is compatible with most of MMSSTV and EasyPal.